src/world.rs

```diff
@@ -297,6 +297,12 @@ impl Generation {
         Generation(v)
     }
 
+    /// Returns the id of the generation.
+    #[inline]
+    pub fn get_id(&self) -> i32 {
```

Comment:
Getters in Rust shouldn't start with `get_` unless necessary. Would recommend renaming it to `generation()`.